app/utils/postgres.js

```diff
@@ -31,6 +31,17 @@ const getUserGames = async (pool, userId)=> {
     }
 }
 
+const getUserNames = async (pool, portfolioId)=> {
```

Comment:
can you change the name to like getUsernameFromPortofolio or something because right now its not very descriptive
also this only gives us one username right?